docs/reference/edgectl-download.md

```diff
@@ -0,0 +1,29 @@
+# Downloads
+
+Edge Control is available for macOS, Linux, and Windows.
+
+## macOS
+
+Download by curl command with:
+
+      ```shell
+      sudo curl -fL https://metriton.datawire.io/downloads/darwin/edgectl -o /usr/local/bin/edgectl && sudo chmod a+x /usr/local/bin/edgectl
+      ```
+
+Or, download the [binary file](https://metriton.datawire.io/downloads/darwin/edgectl).
```

Comment:
Should we mention you need to change the permission if you download the file directly?

Reply:
fixed